src/Raven.Server/Documents/Indexes/Static/JavaScriptIndexUtils.cs

```diff
@@ -88,7 +89,15 @@ public static bool GetValue(Engine engine, object item, out JsValue jsItem, bool
         {
             jsItem = null;
             if (!(item is DynamicBlittableJson dbj))
+            {
+                //This is the case for map-reduce
+                if(item is BlittableJsonReaderObject bjr)
+                {
+                    jsItem = new BlittableObjectInstance(engine, null, bjr, DynamicNullObject.Null, null);
```

Comment:
Why `DynamicNullObject` here?

Reply:
This is the case where this is not a document and we don't have an ID we just have a blittable, what should i have used?